workflow/util/util_test.go

```diff
@@ -2102,3 +2102,367 @@ func TestRetryWorkflowWithNestedDAGsWithSuspendNodes(t *testing.T) {
 	assert.Equal(t, wfv1.NodeSucceeded, wf.Status.Nodes.FindByName("fail-two-nested-dag-suspend.dag1-step4").Phase)
 	assert.Equal(t, 1, len(podsToDelete))
 }
+
+var failedWorkflowWithContinueOn = `
```

Comment:
similar to the e2e test data, we want to clean up the test data here.
see the other example from the same file, https://github.com/argoproj/argo-workflows/blob/main/workflow/util/util_test.go#L766
we want a minimise version. (e.g. the artifactory has no use here)

Reply:
I deleted artifactory related